healthcheck/utils.py

```diff
@@ -0,0 +1,26 @@
+import errno
+import os
+
+
+def file_exists(path):
+    """Return True if a file exists at `path` (even if it can't be read),
+    otherwise False.
+
+    This is different from os.path.isfile and os.path.exists which return
+    False if a file exists but the user doesn't have permission to read it.
```

Comment:
> Surely from the point of view of this check, not being able to access the file is equivalent to it not existing?

Not being able to traverse to the file means that this check could give unexpected results. I think it's better to fail with an error that tells you what is going on than to treat the 2 conditions the same.